scripts/commands/additem.lua

```diff
@@ -16,9 +16,7 @@ end;
 
 function onTrigger(player, itemId, quantity, aug0, aug0val, aug1, aug1val, aug2, aug2val, aug3, aug3val, trialId)
     -- Load needed text ids for players current zone..
-    local TextIDs = "scripts/zones/" .. player:getZoneName() .. "/TextIDs";
-    package.loaded[TextIDs] = nil;
-    require(TextIDs);
+    local ID = zones[player:getZoneID()]
```

Comment:
I think these (ie. all global functions that are getting IDs) should all be `local ID = require("scripts/zones/" .. player:getZoneName() .. "/IDs.lua")` right? if you zone into a zone and immediately `!addItem`, there's no guarantee the text ids table was loaded (it would be relying on any other npc in the zone who'd happen to load them before)

Reply:
The IDs.lua are required by every zone's Zone.lua file, so full table will be built at server start.

Reply:
hmm.. that's good enough then, I think